JoinColony/colonyCDapp
# fix: Add the missing icon to the re-enable extension modal

## Description

| Before | After |
| ---- | ---- |
| ![Screenshot 2024-08-28 at 10 34 23](https://github.com/user-attachments/assets/51daa38c-d2e8-4664-83c6-ebbf9a6ba76a) | ![Screenshot 2024-08-28 at 10 34 28](https://github.com/user-attachments/assets/c0b15d33-b099-453a-b4b0-3f76b5b62b62) |

## Testing

1. Install and enable the Multi-sig extension
2. Deprecate the Multi-sig extension
3. Enable the Multi-sig extension
4. Verify that the Re-enable Extension Modal follows this design pattern:

![image](https://github.com/user-attachments/assets/f921cd73-f016-448a-b5dd-32fae8660227)

Resolves #2776
